src/test/java/seedu/address/ui/CommandHistoryTest.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.ui;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+class CommandHistoryTest {
+
+    private CommandHistory commandHistory = new CommandHistory();
+    private final String[] commands = {"view 1", "delappt 2", "consult 1", "edit 4 n/Charles", "view 1"};
+
+    @Test
+    void addCommand_addFirstCommand_success() {
+        commandHistory.addCommand(commands[0]);
+        commandHistory.previousCommand();
+        assertEquals(commandHistory.getCommand().orElse(""), commands[0]);
+
+        commandHistory = new CommandHistory();
```

Comment:
You can either move this line out or remove this because it does not help with the current test case.